ui/app/components/secret-engine/configure-azure.hbs

```diff
@@ -0,0 +1,107 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form {{on "submit" (perform this.submitForm)}} aria-label="save azure creds" data-test-configure-form>
+
+  <NamespaceReminder @mode="save" @noun="configuration" />
+  <MessageError @errorMessage={{this.errorMessage}} />
+
+  <div class="box is-fullwidth is-sideless">
+    {{! WIF is an enterprise only feature. We default to Azure access type for community users and display only those related form fields. }}
+    {{#if this.version.isEnterprise}}
+      <fieldset class="field form-fieldset" id="protection" data-test-access-type-section>
+        <legend class="is-label">Access Type</legend>
+        <p class="sub-text" data-test-access-type-subtext>
+          {{#if this.disableAccessType}}
+            You cannot edit Access Type if you have already saved access credentials.
+          {{else}}
+            Choose the way to configure access to Azure. Access can be configured either with Azure account, or using Plugin
+            Workload Identity Federation (WIF).
```

Comment:
nit: I feel like this makes a little more sense grammatically? At the very least, I think there `an` should be added before "Azure"
```suggestion
            Choose the way to configure access to Azure. Access can be configured either using an Azure account or with the Plugin
            Workload Identity Federation (WIF).
```